app/src/pages/inside/common/stackTraceMessageBlock/stackTraceMessageBlock.jsx

```diff
@@ -30,12 +30,14 @@ export class StackTraceMessageBlock extends Component {
     children: PropTypes.any,
     maxHeight: PropTypes.number,
     level: PropTypes.string,
+    alternateDesign: PropTypes.bool,
```

Comment:
`alternateDesign` is a good idea, but it might be more flexible to specify which type of `alternateDesign` will be used?
I suggest to add a `designMode` property or something similar with a value, eg. `dark`.